### DISCUSSION TITLE
Iris installer mod for Minecraft Fabric not opening properly.

### DISCUSSION THREAD

**Discussion by S0yS4uce** [community/community#112225] [Programming Help]
Use to open normally in Windows 10 but changed directory in Windows 11. Now it opens in a GIMP directory for some reason even though I deleted GIMP ages ago. 

- Setting it to open on Java just pulls up a blank command prompt which then deletes itself
- The installer version was downloaded after I found out about the problem so it's the most recent version
- (Figure 2) The directory mentioned in the application list and the one that the .jar tried to open with doesn't even exist on my computer, of course it doesn't because I don't have GIMP.

**UPDATE: SOLUTION FOUND WITH JARFIX DOWNLOAD AND RUN COURTESY OF COPILOT'S SUGGESTION**

I've asked various LLMs for answers but the answers were always generalised (not helpful) and searched on forums but there was no situation like mine.
![Never_seen_before](https://github.com/community/community/assets/161796115/5e06dd0c-ebf2-44f4-979c-fa53ff94e006)
Figure 1
![Extra dumb](https://github.com/community/community/assets/161796115/db3186b7-d75c-409a-b193-624109b78065)
Figure 2

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @S0yS4uce ! 

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.
